extensions/amp-redbull-player/0.1/utils/addAttributesToElement.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * Add attributes to an element.
+ * @param {!Element} el
+ * @return {!Element} el
+ */
+const addAttributesToElement = el => {
```

Comment:
Why are we doing this? This is being used to set attributes on the outer `<amp-redbull-player>` element, not the `<iframe>`

Reply:
removed